src/common/prop.c

```diff
@@ -674,6 +699,14 @@ daos_prop_copy(daos_prop_t *prop_req, daos_prop_t *prop_reply)
 				D_GOTO(out, rc);
 
 			roots_alloc = true;
+		} else if (type == DAOS_PROP_PO_POLICY) {
+			pd = entry_req->dpe_val_ptr;
+			D_ALLOC(pd, sizeof(*entry_reply->dpe_val_ptr));
+			if (pd == NULL)
+				D_GOTO(out, rc = -DER_NOMEM);
+
+			memcpy(pd, entry_reply->dpe_val_ptr, sizeof(*entry_reply->dpe_val_ptr));
+			policy_alloc = true;
```

Comment:
This isn't necessary, you can check if pd is NULL later.

Reply:
I think it is not relevant now, as the policy property will be changed to a string, so dpe_val_ptr will not be used in the end.